test/parallel/test-domain-uncaught-exception.js

```diff
@@ -23,7 +23,7 @@ function test1() {
    */
   const d = domain.create();
   d.run(function() {
-    setTimeout(function onTimeout() {
+    setImmediate(function onSetImmediate() {
```

Comment:
The test used to test that exceptions thrown from setTimeout were in the domain, now they test that test thrown from setImmediate are still in the domain! I don't think you are looking at what the test is doing.